src/custom_baudrate.cpp

```diff
@@ -0,0 +1,14 @@
+#include <asm/termios.h>
+
+int ioctl(int fd, int request, ...);
+
+void set_custom_baudrate(const int fd, const int baudrate) {
```

Comment:
I'm not too sharp on C++, I had a friend help make the code compile. In order to have it recognize termios2 we had to include the <asm/termios.h> header. We would have to figure out a way to have both termios together, or on linux have all files call <asm/termios.h> instead of <termios.h>